numpy/__init__.pyi

```diff
@@ -3321,11 +3320,7 @@ class RankWarning(UserWarning): ...
 
 _CallType = TypeVar("_CallType", bound=_ErrFunc | _SupportsWrite[str])
 
-class errstate(Generic[_CallType], ContextDecorator):
-    call: _CallType
-    kwargs: _ErrDictOptional
-
-    # Expand `**kwargs` into explicit keyword-only arguments
+class errstate(Generic[_CallType]):
```

Comment:
```suggestion
class errstate:
```
Not really a reason to keep this generic now with the removal of the `call` attribute.